src/dpp/sslclient.cpp

```diff
@@ -504,6 +504,15 @@ void ssl_client::read_loop()
 							case SSL_ERROR_NONE:
 								/* Data received, add it to the buffer */
 								if (r > 0) {
+									const std::string data(server_to_client_buffer, r);
+									const std::vector<std::string>& data_lines = utility::tokenize(data);
+									const std::string& http_reponse(data_lines[0]);
+
+									/* Does the first line begin with a http code and does it not contain either 204, 101 or 200? */
+									if(http_reponse.rfind("HTTP/1.1", 0) != std::string::npos && http_reponse.find("204") == std::string::npos && http_reponse.find("101") == std::string::npos && http_reponse.find("200") == std::string::npos) {
+										log(ll_warning, "Received unhandled code: " + http_reponse);
```

Comment:
you could just use tokenize again to split by space. this code will fail if anywhere else in the first line has these numbers e.g.

`HTTP/1.1 403 The maximum number of users (200) is exceeded`

convoluted example but you get what I mean yes?

Reply:
I get what you mean yeah :)